draft-ietf-httpbis-digest-headers.md

```diff
@@ -250,11 +251,11 @@ The example below shows the  `sha-256` digest-algorithm which uses base64 encodi
    sha-256=X48E9qOokqqrvdts8nOJRJN3OWDUoyWxBf7kbu9DBPE=
 ~~~
 
-# The Digest Header Field {#digest-header}
+# The Digest Field {#digest-field}
 
-The Digest header field contains a list of one or more representation digest values
-as defined in {{representation-digest}}.
-It can be used in both request and response.
+The Digest field contains a list of one or more representation digest values as
+defined in {{representation-digest}}. It can be used in both request and
+response.
```

Comment:
```suggestion
The Digest field contains a list of one or more representation digest values
as defined in {{representation-digest}}.
It can be used in both request and response.
```

Reply:
I'm reflowing paragraphs all over the doc in this PR. This one has bad source formatting, so why not change it?